api/src/main/java/org/openmrs/module/aijar/dataintegrity/InvalidARTEncounters.java

```diff
@@ -117,7 +117,7 @@
 	 */
 	private List<RuleResult<Patient>> patientsWithMoreThanOneEncounterOnTheSameDate() {
 		log.info("Executing rule to find patients with more than one encounters on a single date");
-		String queryString = "SELECT visit FROM Encounter e GROUP BY e.encounterType.encounterTypeId, e.visit.visitId HAVING count(e.visit.visitId) > 1";
+		String queryString = "SELECT visit FROM Encounter e WHERE e.encounterType.uuid='8d5b2be0-c2cc-11de-8d13-0010c6dffd0f' GROUP BY e.encounterType.encounterTypeId, e.visit.visitId HAVING count(e.visit.visitId) > 1";
```

Comment:
@slubwama add checing for encounters that are not voided (especially when deleted) 

Can you also look at all the other data validation queries to ensure this is taken care of (the voiding I mean)